src/webchat-ui/components/plugins/input/base/BaseInput.tsx

```diff
@@ -391,12 +393,20 @@ export class BaseInput extends React.PureComponent<IBaseInputProps, IBaseInputSt
 		const { text, textActive, speechResult: speechInterim } = state;
 
 		const {
-			disableInputAutocomplete,
-			disableInputAutofocus,
+			layout,
 			enableFileAttachment,
```

Comment:
This is different now `fileStorageSettings.enabled`. We also need to handle the new setting `fileStorageSettings.dropzoneText`

Reply:
Also `businessHours.timeZone `should be changed to` businessHours.timeyone`

Reply:
hm there was enableFileUpload and enableFileAttachment, not sure what the difference is but the spec says to replace enableFileUpload. I think we need to clarify this.

Reply:
after investigating, I removed both old settings mentioned above and implemented it to use the two filestoragesettings keys